ticker.go

```diff
@@ -36,3 +48,26 @@ func (t *Ticker) UnmarshalJSON(data []byte) error {
 	}
 	return nil
 }
+
+func (ts Tickers) UnmarshalJSON(data []byte) error {
+	var arr []json.RawMessage
+
+	if err := json.Unmarshal(data, &arr); err != nil {
+		return err
+	}
+
+	for _, v := range arr {
+		var ticker Ticker
+
+		err := json.Unmarshal(v, &ticker)
+		if ticker.Volume == 0 {
```

Comment:
sould not filter out if volume is zero

Reply:
If volume is 0 the market is inactive
This is why it is filtered out

Reply:
inactive means, "disabled" by hitbtc, or no trades on that day?

Reply:
Inactive, I checked it by running tests

Reply:
Sorry disabled

Reply:
hmm, than, this may hide the "enabled but no trades today" market from list :/